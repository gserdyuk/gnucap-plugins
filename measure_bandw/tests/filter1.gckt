* rc filter
spice
v1 1 0 ac 1
r1 1 2 10k
c1 2 0 1u
rl 2 0 10k

.store ac vdb(2) vdb(1)
.print ac vdb(2) vdb(1)
.ac oct 10 1 1000k
.measure band=bandwidth("vdb(2)",3)
.end
