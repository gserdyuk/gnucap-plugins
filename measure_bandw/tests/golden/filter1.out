Gnucap gnucap-arails  2009.12.07 RCS 26.136 master 2010-04-09
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
#Freq       vdb(2)     vdb(1)    
 1.        -6.0249    -4.3472n   
 1.0718    -6.0255    -4.3479n   
 1.1487    -6.0263    -4.3486n   
 1.2311    -6.0271    -4.3494n   
 1.3195    -6.0281    -4.3504n   
 1.4142    -6.0292    -4.3515n   
 1.5157    -6.0304    -4.3528n   
 1.6245    -6.0319    -4.3542n   
 1.7411    -6.0336    -4.3559n   
 1.8661    -6.0355    -4.3578n   
 2.        -6.0377    -4.36n     
 2.1435    -6.0403    -4.3626n   
 2.2974    -6.0432    -4.3655n   
 2.4623    -6.0465    -4.3688n   
 2.639     -6.0503    -4.3726n   
 2.8284    -6.0548    -4.377n    
 3.0314    -6.0598    -4.382n    
 3.249     -6.0656    -4.3877n   
 3.4822    -6.0723    -4.3943n   
 3.7321    -6.0799    -4.4018n   
 4.        -6.0886    -4.4105n   
 4.2871    -6.0987    -4.4203n   
 4.5948    -6.1102    -4.4316n   
 4.9246    -6.1233    -4.4445n   
 5.278     -6.1384    -4.4592n   
 5.6569    -6.1556    -4.4759n   
 6.0629    -6.1754    -4.495n    
 6.498     -6.1979    -4.5167n   
 6.9644    -6.2237    -4.5413n   
 7.4643    -6.2531    -4.5693n   
 8.        -6.2866    -4.601n    
 8.5742    -6.3248    -4.6367n   
 9.1896    -6.3683    -4.6771n   
 9.8492    -6.4177    -4.7224n   
 10.556    -6.4737    -4.7732n   
 11.314    -6.5373    -4.8301n   
 12.126    -6.6091    -4.8933n   
 12.996    -6.6902    -4.9635n   
 13.929    -6.7815    -5.0409n   
 14.929    -6.884     -5.126n    
 16.       -6.9989    -5.2189n   
 17.148    -7.1273    -5.3199n   
 18.379    -7.2702    -5.4288n   
 19.698    -7.4287    -5.5456n   
 21.112    -7.604     -5.6698n   
 22.627    -7.7969    -5.8008n   
 24.251    -8.0084    -5.938n    
 25.992    -8.2394    -6.0803n   
 27.858    -8.4903    -6.2266n   
 29.857    -8.7618    -6.3756n   
 32.       -9.054     -6.5259n   
 34.297    -9.367     -6.6761n   
 36.758    -9.7008    -6.8248n   
 39.397    -10.055    -6.9706n   
 42.224    -10.429    -7.1122n   
 45.255    -10.823    -7.2484n   
 48.503    -11.234    -7.3785n   
 51.984    -11.664    -7.5016n   
 55.715    -12.11     -7.6172n   
 59.714    -12.571    -7.7249n   
 64.       -13.047    -7.8246n   
 68.594    -13.536    -7.9164n   
 73.517    -14.038    -8.0003n   
 78.793    -14.55     -8.0766n   
 84.449    -15.072    -8.1456n   
 90.51     -15.604    -8.2079n   
 97.006    -16.144    -8.2637n   
 103.97    -16.691    -8.3137n   
 111.43    -17.244    -8.3582n   
 119.43    -17.804    -8.3978n   
 128.      -18.368    -8.433n    
 137.19    -18.938    -8.464n    
 147.03    -19.511    -8.4915n   
 157.59    -20.088    -8.5156n   
 168.9     -20.668    -8.5369n   
 181.02    -21.25     -8.5556n   
 194.01    -21.836    -8.5721n   
 207.94    -22.423    -8.5864n   
 222.86    -23.012    -8.5991n   
 238.86    -23.603    -8.6101n   
 256.      -24.195    -8.6198n   
 274.37    -24.789    -8.6282n   
 294.07    -25.383    -8.6356n   
 315.17    -25.979    -8.642n    
 337.79    -26.575    -8.6477n   
 362.04    -27.172    -8.6526n   
 388.02    -27.77     -8.6569n   
 415.87    -28.368    -8.6606n   
 445.72    -28.967    -8.6639n   
 477.71    -29.566    -8.6667n   
 512.      -30.166    -8.6692n   
 548.75    -30.766    -8.6713n   
 588.13    -31.366    -8.6732n   
 630.35    -31.966    -8.6748n   
 675.59    -32.567    -8.6763n   
 724.08    -33.168    -8.6775n   
 776.05    -33.769    -8.6786n   
 831.75    -34.37     -8.6795n   
 891.44    -34.971    -8.6804n   
 955.43    -35.572    -8.6811n   
 1.024K    -36.174    -8.6817n   
 1.0975K   -36.775    -8.6822n   
 1.1763K   -37.377    -8.6827n   
 1.2607K   -37.979    -8.6831n   
 1.3512K   -38.58     -8.6835n   
 1.4482K   -39.182    -8.6838n   
 1.5521K   -39.784    -8.6841n   
 1.6635K   -40.386    -8.6843n   
 1.7829K   -40.987    -8.6845n   
 1.9109K   -41.589    -8.6847n   
 2.048K    -42.191    -8.6848n   
 2.195K    -42.793    -8.685n    
 2.3525K   -43.395    -8.6851n   
 2.5214K   -43.997    -8.6852n   
 2.7024K   -44.599    -8.6853n   
 2.8963K   -45.201    -8.6854n   
 3.1042K   -45.803    -8.6854n   
 3.327K    -46.405    -8.6855n   
 3.5658K   -47.007    -8.6855n   
 3.8217K   -47.609    -8.6856n   
 4.096K    -48.211    -8.6856n   
 4.39K     -48.813    -8.6857n   
 4.7051K   -49.415    -8.6857n   
 5.0428K   -50.017    -8.6857n   
 5.4047K   -50.619    -8.6857n   
 5.7926K   -51.221    -8.6858n   
 6.2084K   -51.823    -8.6858n   
 6.654K    -52.425    -8.6858n   
 7.1316K   -53.027    -8.6858n   
 7.6434K   -53.629    -8.6858n   
 8.192K    -54.231    -8.6858n   
 8.78K     -54.834    -8.6858n   
 9.4101K   -55.436    -8.6858n   
 10.086K   -56.038    -8.6858n   
 10.809K   -56.64     -8.6859n   
 11.585K   -57.242    -8.6859n   
 12.417K   -57.844    -8.6859n   
 13.308K   -58.446    -8.6859n   
 14.263K   -59.048    -8.6859n   
 15.287K   -59.65     -8.6859n   
 16.384K   -60.252    -8.6859n   
 17.56K    -60.854    -8.6859n   
 18.82K    -61.456    -8.6859n   
 20.171K   -62.058    -8.6859n   
 21.619K   -62.66     -8.6859n   
 23.17K    -63.262    -8.6859n   
 24.834K   -63.864    -8.6859n   
 26.616K   -64.466    -8.6859n   
 28.526K   -65.068    -8.6859n   
 30.574K   -65.671    -8.6859n   
 32.768K   -66.273    -8.6859n   
 35.12K    -66.875    -8.6859n   
 37.641K   -67.477    -8.6859n   
 40.342K   -68.079    -8.6859n   
 43.238K   -68.681    -8.6859n   
 46.341K   -69.283    -8.6859n   
 49.667K   -69.885    -8.6859n   
 53.232K   -70.487    -8.6859n   
 57.052K   -71.089    -8.6859n   
 61.147K   -71.691    -8.6859n   
 65.536K   -72.293    -8.6859n   
 70.24K    -72.895    -8.6859n   
 75.281K   -73.497    -8.6859n   
 80.684K   -74.099    -8.6859n   
 86.475K   -74.701    -8.6859n   
 92.682K   -75.303    -8.6859n   
 99.334K   -75.906    -8.6859n   
 106.46K   -76.508    -8.6859n   
 114.1K    -77.11     -8.6859n   
 122.29K   -77.712    -8.6859n   
 131.07K   -78.314    -8.6859n   
 140.48K   -78.916    -8.6859n   
 150.56K   -79.518    -8.6859n   
 161.37K   -80.12     -8.6859n   
 172.95K   -80.722    -8.6859n   
 185.36K   -81.324    -8.6859n   
 198.67K   -81.926    -8.6859n   
 212.93K   -82.528    -8.6859n   
 228.21K   -83.13     -8.6859n   
 244.59K   -83.732    -8.6859n   
 262.14K   -84.334    -8.6859n   
 280.96K   -84.936    -8.6859n   
 301.12K   -85.539    -8.6859n   
 322.74K   -86.141    -8.6859n   
 345.9K    -86.743    -8.6859n   
 370.73K   -87.345    -8.6859n   
 397.34K   -87.947    -8.6859n   
 425.85K   -88.549    -8.6859n   
 456.42K   -89.151    -8.6859n   
 489.18K   -89.753    -8.6859n   
 524.29K   -90.355    -8.6859n   
 561.92K   -90.957    -8.6859n   
 602.25K   -91.559    -8.6859n   
 645.47K   -92.161    -8.6859n   
 691.8K    -92.763    -8.6859n   
 741.46K   -93.365    -8.6859n   
 794.67K   -93.967    -8.6859n   
 851.71K   -94.569    -8.6859n   
 912.84K   -95.171    -8.6859n   
 978.36K   -95.774    -8.6859n   
 1.0486Meg -96.376    -8.6859n   
band= 28.85706
